__tests__/components/TableDisplay.spec.ts

```diff
@@ -0,0 +1,1480 @@
+                    },
+                ],
+            },
+            {
+                assetType: 'namespace',
+                state: { ownedNamespaces: mockNamespaces, currentHeight: 612920 },
+                expectedResult: [
+                    {
+                        hexId: '80DE90A24D6C0CC4',
+                        name: 'helloworld',
+                        expiration: '2 d 3 h 0 m ',
+                        expired: false,
+                        aliasType: 'N/A',
+                        aliasIdentifier: 'N/A',
+                        metadataList: [],
+                    },
+                    {
+                        hexId: '80DE90A24D6C0CC4',
+                        name: 'city.access',
+                        expiration: '2 d 3 h 0 m ',
+                        expired: false,
+                        aliasType: 'mosaic',
+                        aliasIdentifier: '632B3D23282A51E8',
+                        metadataList: [],
+                    },
+                    {
+                        hexId: '80DE90A24D6C0CC4',
+                        name: 'symbolcity',
+                        expiration: '2 d 3 h 0 m ',
+                        expired: false,
+                        aliasType: 'address',
+                        aliasIdentifier: 'TCABUWAK5WMJ26ZPERMGWBOWAJF4XPNCJOWPAAI',
+                        metadataList: [],
+                    },
+                ],
+            },
+            {
+                assetType: 'metadata',
+                state: { accountMetadataList: mockMetadatas },
+                expectedResult: [
+                    {
+                        hexId: '62FC80CAE173875059E7274E',
+                        scopedMetadataKey: 'E3E5E7B070607991',
+                        targetAddress: 'TCABUWAK5WMJ26ZPERMGWBOWAJF4XPNCJOWPAAI',
+                        targetId: '80DE90A24D6C0CC4',
+                        value: 'metadata',
+                    },
+                    {
+                        hexId: '6303F2DC6CF8F07F3107CEB9',
+                        scopedMetadataKey: 'B660AB4DEF66DFFA',
+                        targetAddress: 'TCABUWAK5WMJ26ZPERMGWBOWAJF4XPNCJOWPAAI',
+                        targetId: '4151C48416E93B75',
+                        value: 'metadata',
+                    },
+                ],
+            },
+        ];
+
+        rows.forEach(({ assetType, state, expectedResult }) => runBasicTableRowsTests(assetType, state, expectedResult));
+    });
+
+    describe('displayedValues', () => {
+        const runBasicFilteredDisplayedValuesTests = (assetType, params, expectedDefaultResult, expectedExpiredResult) => {
+            // Arrange:
+            const wrapper = getTableRowWrapper(
+                { ...params },
+                {
+                    assetType,
+                },
+            );
+
+            const vm = wrapper.vm as TableDisplayTs;
+
+            test(`returns default ${assetType} when filter expiration is hide`, () => {
```

Comment:
in this case it shows both expired and unexpired, so test name is slightly misleading